### PR TITLE
Added outfit list and outfit cards

### PR DESCRIPTION
**Pull request by glyphgriff**
clicking the + on the first card in the outfit list will add another card to the list. Eventually (tomorrow) each new card will be populated with the overview item's properties.

### CODE REVIEW

File: client/src/components/rc/OutfitCard.jsx [hunk 1]
```diff
@@ -1,9 +1,62 @@
...
+  const [photoData, setPhotoData] = useState([]);
+  const [price, setPrice] = useState([]);
+  const [starRating, setStarRating] = useState(0);
+
+  useEffect(() => {
+    request
+      .getStyles(productID)
+      .then((data) => {
+        // console.log('styles data:', data.data);
+        setPhotoData(data.data.results[0].photos[0].url);
```
Comment (thachdo90) [inline client/src/components/rc/OutfitCard.jsx:19]: consider aliasing data.data.results[0] so you don't have to repeat that in 3 places

File: client/src/components/rc/OutfitList.jsx [hunk 2]
```diff
@@ -1,12 +1,53 @@
...
+    setOutfit(prevOutfit => [...prevOutfit, cardCount]);
+  }
+
+  console.log('cardCounter:', cardCount);
+  console.log('outfit:', outfit);
+
+  //add a function that deletes a card when X button is clicked -- delete from state
+  const deleteCard = () => {
+    setOutfit([...outfit.splice(cardCount - 1, 1)]);
+  }
```
Comment (thachdo90) [inline client/src/components/rc/OutfitList.jsx:36]: How does this logic delete the card being clicked instead of just the last one?

**Review by thachdo90 — APPROVED** [review]
Looks good! I had one comment for aliasing a repeated variable
and one question for my own understanding of how your code works to delete the clicked card.

File: client/src/components/rc/OutfitCard.jsx (hunk 1 above)
Comment (glyphgriff) [reply client/src/components/rc/OutfitCard.jsx:19]: Good idea!

File: client/src/components/rc/OutfitList.jsx (hunk 2 above)
Comment (glyphgriff) [reply client/src/components/rc/OutfitList.jsx:36]: It doesn't yet haha. It's supposed to delete the current card from the outfit state array but it's not working so I'm figuring something else out

**Pull request merged**